nestkernel/conn_builder.h

```diff
@@ -87,6 +87,11 @@ class ConnBuilder
   //! Create connection between given nodes, fill parameter values
   void single_connect_( index, Node&, thread, librandom::RngPtr& );
 
+  //! Calls value-function of all parameters being instantiations of ArrayDoubleParameter or
```

Comment:
This comment is not in proper doxygen format (http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html). The common format for multiline Doxygen comments in NEST is

```
/**
 *
 *
 */
```

The first line should be a short summary of the function, followed by an empty line and a more detailed explanation.